serenata_toolbox/ceap_dataset.py

```diff
@@ -1,24 +1,23 @@
 import os.path
+from datetime import date
 from urllib.request import urlretrieve
 from zipfile import ZipFile
 import numpy as np
 import pandas as pd
 from .reimbursements import Reimbursements
-from .xml2csv import convert_xml_to_csv
 
 class CEAPDataset:
     def __init__(self, path):
         self.path = path
 
 
     def fetch(self):
-        urls = ['http://www.camara.gov.br/cotas/AnoAtual.zip',
-                'http://www.camara.gov.br/cotas/AnoAnterior.zip',
-                'http://www.camara.gov.br/cotas/AnosAnteriores.zip']
-        filenames = map(lambda url: url.split('/')[-1], urls)
+        base_url = "http://www.camara.leg.br/cotas/Ano-{}.csv.zip"
 
-        for url, filename in zip(urls, filenames):
-            zip_file_path = os.path.join(self.path, filename)
+        years = [n for n in range(2009, date.today().year+1)]
```

Comment:
We could have a constant `YEARS = [n for n in range(2009, date.today().year+1)]` in the beginning of the file, couldn't we? (relevant here and on line 37)